src/backend/distributed/transaction/lock_graph.c

```diff
@@ -134,7 +134,7 @@ BuildGlobalWaitGraph(void)
 		querySent = SendRemoteCommand(connection, command);
 		if (querySent == 0)
 		{
-			ReportConnectionError(connection, ERROR);
+			ReportConnectionError(connection, WARNING);
```

Comment:
Pretty minor, maybe add `continue;` here as well in case anyone adds some code below?